backend/package.json

```diff
@@ -31,6 +31,7 @@
     "axios": "^0.19.2",
     "bcryptjs": "^2.4.3",
     "cors": "^2.8.5",
+    "datejs": "1.0.0-rc3",
```

Comment:
What about using a more "popular" date library? I think both [date-fns](https://www.npmjs.com/package/date-fns) and [moment](https://www.npmjs.com/package/moment) are great alternatives!

Datejs haven't seen a release in the past 5 years